### ISSUE TITLE
Default vm.args

### ISSUE THREAD

**Issue by varnerac** [vernemq/vernemq#1732]
### To help us save time and help you faster

Should we add a default `vm.args` to the `relx` config? I know that some of the variables are added from the `.conf` file. But, it seems like some sensible defaults around stuff like `ERL_MAX_PORTS`?

### Expected behaviour

- Provide a solid starting point for vals passed to `erl` on startup.

### Actual behaviour

- I *think* in a basic release we just get the stuff from the `.conf` file

**Comment by ioolkos** [comment]
Yeah, we get the stuff from the `.conf` file plus the Erlang defaults. 
What would be additional sensible starting values for `erl`?

Note the following (an existing `vm.args` file overrides the auto-generated ones):
https://docs.vernemq.com/guides/not-a-tuning-guide#the-vm-args-file

**Comment by varnerac** [comment]
Maybe we put some sensible defaults in the conf or allow the conf to update the vm.args?

**Comment by ioolkos** [comment]
Those VM config values end up in the `vm.args` file via Cuttlefish. So the conf can already update the `vm.args` but only for values defined in the Cuttlefish schema:
https://github.com/kyorai/cuttlefish/blob/4f436ba2b67a7a95ed1d4ddd74a44a06661a77d1/priv/erlang_vm.schema
We could probably add more.
